framework/encryption/__init__.py

```diff
@@ -6,13 +6,22 @@
 
 SENSITIVE_DATA_KEY = jwe.kdf(settings.SENSITIVE_DATA_SECRET.encode('utf-8'), settings.SENSITIVE_DATA_SALT.encode('utf-8'))
 
+def ensure_str(value):
+    """Helper function to ensure all inputs are encoded to the proper value utf-8 value regardless of input type"""
+    try:
+        return value.encode('utf-8')
+    except UnicodeDecodeError:
+        return value.decode('utf-8').encode('utf-8')
+
 def encrypt(value):
     if value:
+        value = ensure_str(value)
         return jwe.encrypt(bytes(value), SENSITIVE_DATA_KEY)
     return None
 
 def decrypt(value):
     if value:
+        value = ensure_str(value)
```

Comment:
Here, `value` is going to be an encrypted string of the form
```
r'\w{43}\.\.\w{22}\.\w*\.\w{22}'
```
I'm not sure that this needs to be called here.